src/app/application.h

```diff
@@ -153,7 +162,8 @@ private slots:
 
     QTranslator m_qtTranslator;
     QTranslator m_translator;
-    QStringList m_paramsQueue;
+
+    QList<Params> m_paramsQueue;
```

Comment:
Currently we haven't dropped Qt5, should we still prefer `QVector`?

Reply:
> Currently we haven't dropped Qt5, should we still prefer `QVector`?

It was a list before that, so I don't see the point in changing it on the threshold of Qt6. We don't even know its size to reserve.